README.md

```diff
@@ -62,49 +93,114 @@ information in the console at
 Manage > Access (IAM) > Access groups > Access policies.
 -->
 
-<!--
 You need the following permissions to run this module.
 
 - Account Management
-    - **Sample Account Service** service
-        - `Editor` platform access
-        - `Manager` service access
+    - **All Account Management** services
+        - `Administrator` platform access
     - IAM Services
-        - **Sample Cloud Service** service
+        - **All Identity and Access enabled** services
             - `Administrator` platform access
--->
-
-<!-- NO PERMISSIONS FOR MODULE
-If no permissions are required for the module, uncomment the following
-statement instead the previous block.
--->
-
-<!-- No permissions are needed to run this module.-->
-
+            - `Manager` service access
```

Comment:
what about atracker and cos ? (I added what I though was needed in the ibm_catalog.json in https://github.com/terraform-ibm-modules/terraform-ibm-base-account-enterprise/pull/22)